### PR TITLE
ksmbd-tools: remove unused values

### PR DESCRIPTION
**Pull request by neheb**
Found with clang-analyzer

Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

File: mountd/ipc.c [hunk 1]
```diff
@@ -65,16 +65,13 @@ static int generic_event(int type, void *payload, size_t sz)
 
 static int parse_reload_configs(const char *pwddb, const char *smbconf)
 {
-	int ret;
-
 	pr_debug("Reload config\n");
 
 	usm_remove_all_users();
-	ret = cp_parse_pwddb(pwddb);
+	int ret = cp_parse_pwddb(pwddb);
```
Comment (namjaejeon) [inline mountd/ipc.c:71]: Doesn't this build warning? I think the original code is correct...

Comment (atheik) [reply mountd/ipc.c:71]: @neheb,

In this codebase, declarations are always at the top of the block. Projects may choose this convention even when they don't limit themselves to ISO C90.

Comment (neheb) [reply mountd/ipc.c:71]: @namjaejeon why would there be a warning?

**Comment by neheb** [comment]
Applied suggested changes.

File: mountd/ipc.c (hunk 1 above)
Comment (neheb) [reply mountd/ipc.c]: Yeah this is fixed.

**Comment by namjaejeon** [comment]
@neheb Applied, Thanks for your patch!

**Pull request closed without merging**